### PR TITLE
CSV uploader

### PR DESCRIPTION
**Pull request by lhuygi**
Allows uploading multiple CSV files, e.g.,

    curl -H Authorization:<some uuid> -F vreName=mycollection \
        -F 'file=@persons.csv;filename=persons.csv' \
        -F 'file=@orgs.csv;filename=orgs.csv' \
        http://localhost:8080/v2.1/bulk-upload/

### CODE REVIEW

**Comment by renevanderark** [comment]
tested with semi-colon separated csv, did not recognise columns, is this feature easy to implement?

**Comment by lhuygi** [comment]
You need to change the CSVFormat used inside the CsvLoader and tell the REST service about which format is being sent. It's easy but takes some thought in terms of what the API should be, given that the number of CSV variants is endless.

**Comment by jauco** [comment]
I have somenunpushed commits. One of em adds the ability to specify csv parsing options in the http request (as form fields)

**Comment by renevanderark** [comment]
alright, let's resolve it through form variables... working on multi-upload first (https://github.com/HuygensING/timbuctoo-default-frontend/tree/import-types)

**Comment by jauco** [comment]
@lhuygi I added a few commits. Some where stylistic, some added new features. Are you okay with them? @renevanderark at https://github.com/HuygensING/timbuctoo/pull/140/commits/f63a84138285936c6927ab73d72bd23d12306e56#diff-1a2bce53f603cc6f1b96a3c8a24f8f3fR26 you can find the expected property names for csv parser config

File: timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/bulkupload/loaders/csv/CsvLoader.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * A CsvLoader parses CSV files and treats each file as a separate collection.
+ * The default format is Excel's CSV format, with a header row.
+ */
+public class CsvLoader implements Loader {
+  private final CSVFormat format;
+
+  public CsvLoader(Map<String, String> config) {
+    CSVFormat format = CSVFormat.EXCEL;
+    if (config.containsKey("delimiter")) {
+      format = format.withDelimiter(config.get("delimiter").charAt(0));
```
Comment (renevanderark) [inline timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/bulkupload/loaders/csv/CsvLoader.java]: yes, that would be safer in all cases... I had that typed in but reverted it due to a temporary lack of pro-activity on my part.

**Pull request merged**